firmware/controllers/bench_test.cpp

```diff
@@ -129,153 +126,153 @@ static int count;
 static brain_pin_e brainPin;
 static OutputPin* pinX;
 
-static void pinbench(const char *delayStr, const char *onTimeStr, const char *offTimeStr, const char *countStr,
-		OutputPin* pinParam, brain_pin_e brainPinParam) {
-	delayMs = atoff(delayStr);
-	onTime = atoff(onTimeStr);
-	offTime = atoff(offTimeStr);
-	count = atoi(countStr);
-
-	brainPin = brainPinParam;
+static void pinbench(float dly, float on, float off, int cnt,
```

Comment:
no contractions of short words please

Reply:
All self-explaining names are used by static globals

Reply:
```
struct state
{
    int count;
    float on;
    .......
};
state benchState;



{
    benchState.count = count;
    benchState.on = on;
     ...........
}
```

Reply:
`state` can be merged to `BenchController`. But lets do it in next iteration.